package.json

```diff
@@ -54,27 +54,25 @@
   "license": "UNLICENSED",
   "types": "./ts/index.d.ts",
   "peerDependencies": {
-    "@xstyled/styled-components": "^1.15.1",
-    "@xstyled/system": "^1.15.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "styled-components": "^4.4.0"
+    "styled-components": "^4.4.0 || ^5.0.0",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
   },
   "devDependencies": {
     "@babel/core": "^7.5.5",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/plugin-proposal-export-default-from": "^7.5.2",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-typescript": "^7.3.3",
-    "@storybook/addon-a11y": "^5.3.3",
-    "@storybook/addon-actions": "^5.3.3",
-    "@storybook/addon-info": "^5.3.3",
-    "@storybook/addon-knobs": "^5.3.3",
-    "@storybook/addon-links": "^5.3.3",
-    "@storybook/addon-options": "^5.3.3",
-    "@storybook/addon-storyshots": "^5.3.3",
-    "@storybook/addons": "^5.3.3",
-    "@storybook/react": "^5.3.3",
+    "@storybook/addon-a11y": "^5.3.12",
```

Comment:
hotfix `^5.3.12` resolves the `.toJson` issue

Reply:
Oh that's awesome. Very quick turnaround.